tooling/nargo_cli/tests/execution_success/trait_self/src/main.nr

```diff
@@ -1,5 +1,13 @@
 trait ATrait {
     fn asd() -> Self;
+
+    fn static_method() -> Field {
+        Self::static_method_2()
+    };
```

Comment:
I missed that we currently require semicolons after default functions in a trait. These should be optional eventually but it's not a priority.

Reply:
Added it to the roadmap